### DISCUSSION TITLE
Importance of Github Copilot

### DISCUSSION THREAD

**Discussion by GeeksforGeeksRec** [community/community#107527] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Lets discuss regarding Github Copilot.

**Comment by raghavnallana17298** [comment, marked as answer]
Github copilot is going to be one of the important thing as its going to increase the speed of a coding procedure by providing few code snippets/suggestions. This can help the developers to save time and increase the efficiency.

**Comment by harshavardhanyadav2004** [comment]
GitHub copilot is something like which is mainly AI Pair programmer that give suggestions as per your code .We can generally receive suggestions from the github Copilot by starting to write the code or it allows us to the write the code and also allows us to take natural language from the user to get the code ..
